api-client/src/main/java/com/takipi/api/client/result/event/MainStats.java

```diff
@@ -0,0 +1,28 @@
+package com.takipi.api.client.result.event;
+
+import com.google.common.collect.Lists;
+import com.takipi.api.client.data.event.Stats;
+
+import java.util.List;
+
+public class MainStats extends ApiBaseStats {
```

Comment:
MainStats doesn't describe what kind of stats this is, especially when comparing to Stats.
I would consider name changes to this class and maybe to Stats as well

Reply:
I adjusted it to the name of the proto MainEventStats